src/main/java/com/openlattice/datastore/data/controllers/DataController.java

```diff
@@ -1117,6 +1119,10 @@ private WriteEvent clearOrDeleteEntities( UUID entitySetId, Set<UUID> entityKeyI
         return new WriteEvent( maxVersion, numUpdates );
     }
 
+    private void performAssociationAccessChecksForDelete( UUID entitySetId, DeleteType deleteType ) {
+        // TODO
```

Comment:
Just curious: what access checks are missing here? We do some checks in `deleteAssociations`

Reply:
oh whoops, didn't realize that was a local change. removing...